packages/py-api/py_api/models/hackathon_participants_models.py

```diff
@@ -0,0 +1,8 @@
+from pydantic import BaseModel
+
+
+class RandomParticipant(BaseModel):
+    first_name: str
+    last_name: str
+    email: str
+    Tshirt_size: str
```

Comment:
Rename this to tshirt_size or shirt_size. Snake case is the convention for naming variables and functions in Python.

Reply:
Refer to this when defining the type of t-shirts. 
https://stackoverflow.com/a/65680510

```python
TSHIRT_SIZES=Literal["S", "M", "L"]

...

class Whatever(BaseModel):
    tshirt_sizes: TSHIRT_SIZES
```